CODEOWNERS: add more team members

andy-liu-telink opened the pull request:
- Added s07641069, andriy-bilynskyy, and mishadesh as code owners
- Added andriy-bilynskyy and mishadesh to hal_telink as collaborators
Signed-off-by: Andy Liu <jie.liu@telink-semi.com>

andy-liu-telink commented:
@stephanosio 
Could you approve this PR?

stephanosio reviewed (commented):
CODEOWNERS will be deprecated soon. Please add these users to the MAINTAINERS file (for now, it is ok to add them in both).

andy-liu-telink committed:
CODEOWNERS: add more team members

Added s07641069, andriy-bilynskyy, and mishadesh as code owners
Added andriy-bilynskyy and mishadesh to hal_telink as collaborators

Signed-off-by: Andy Liu <jie.liu@telink-semi.com>

andy-liu-telink commented:
@stephanosio 
I have added them to the MAINTAINERS file.

andy-liu-telink commented:
@MaureenHelm 
Could you approve this pull request?

andy-liu-telink commented:
@stephanosio @MaureenHelm
Can both of you approve this PR? Thanks.

stephanosio reviewed (commented):
Please split this into two commits, one for CODEOWNERS and another for MAINTAINERS.
